langchain_mcp_adapters/tools.py

```diff
@@ -253,4 +258,34 @@ async def fn(**arguments: dict[str, Any]) -> Any:  # noqa: ANN401
         parameters=parameters,
         fn_metadata=fn_metadata,
         is_async=True,
+        context_kwarg=None,
+        annotations=None,
     )
+
+
+@overload
+def to_fastmcp(tool: BaseTool) -> FastMCPTool: ...
+
+
+@overload
+def to_fastmcp(tool: list[BaseTool]) -> list[FastMCPTool]: ...
+
+
+def to_fastmcp(
```

Comment:
I prefer not to do this at least for now, we may discover that we need to pass additional configuration to parameterize how the tool is exposed

```python
to_fastmcp(tool, some configuration)
```